apps/bare-expo/scripts/start-android-e2e-test.js

```diff
@@ -39,7 +40,16 @@ const MAESTRO_DRIVER_STARTUP_TIMEOUT = '120000'; // Wait 2 minutes for Maestro d
       if (!deviceId) {
         throw new Error(`No connected Android device found`);
       }
-      await retryAsync(() => testAsync(projectRoot, deviceId, appBinaryPath, adbPath), 6);
+      await createMaestroFlowAsync({
+        appId: APP_ID,
+        workflowFile: getMaestroFlowFilePath(projectRoot),
+        confirmFirstRunPrompt: true,
+      });
```

Comment:
```suggestion
      const maestroFlowFilePath = await createMaestroFlowAsync({
        appId: APP_ID,
        workflowFile: path.join(projectRoot, MAESTRO_GENERATED_FLOW),
        confirmFirstRunPrompt: true,
      });
```

and pass the `maestroFlowFilePath` into `testAsync`

feel unclear between `createMaestroFlowAsync` / `getMaestroFlowFilePath`. if you want to prevent creating yaml multiple time, you can just pass the returned `maestroFlowFilePath` into `testAsync`. then we don't have to introduce the new getMaestroFlowFilePath function